src/commands/flags.ts

```diff
@@ -578,6 +578,17 @@ export class Flags {
     prompt: undefined,
   };
 
+  public static readonly blockLocalTag: CommandFlag = {
+    constName: 'blockLocalTag',
+    name: 'block-local-tag',
+    definition: {
+      describe: 'Version of local built block node docker image',
```

Comment:
```suggestion
  public static readonly imageTag: CommandFlag = {
    constName: 'imageTag',
    name: 'image-tag',
    definition: {
      describe: 'The Docker image tag to override what is in the Helm Chart',
```
As we move closer to our new CLI presentation design, we want to make sure all of the commands and options are consistent and reusable.